packaging/arch/PKGBUILD

```diff
@@ -3,28 +3,34 @@
 # Contributor: Zygmunt Krynicki <me at zygoon dot pl>
 
 pkgbase=snapd
-pkgname=(snapd snap-confine)
-pkgver=2.26.1
+pkgname=snapd
+pkgver=2.26.14.r1445.g3df03f98b
```

Comment:
Why do we hadd the git revision here? Should this be just plain 2.26.14 or better 2.27 as that is the upcoming next version?